lib/backend/k8s/k8s.go

```diff
@@ -258,6 +275,8 @@ func (c *KubeClient) Create(d *model.KVPair) (*model.KVPair, error) {
 	switch d.Key.(type) {
 	case model.GlobalConfigKey:
 		return c.createGlobalConfig(d)
+	case model.IPPoolKey:
```

Comment:
Is there much of a cost in creating a client?  Just wondering if ipPoolClient should be a function which only instantiates a client when necessary.

`c.ipPools